rest-service/manager_rest/config.py

```diff
@@ -182,6 +183,37 @@ def load_from_db(self):
         # disallow implicit loading
         self.can_load_from_db = False
 
+    def update_db(self, config_dict):
+        """Update a config value.
+
+            Settings which have is_editable set to False, can only be edited
+            when passing the force flag.
+            If a schema is specified for the given setting, the new value
+            must validate.
+            """
+        from datetime import datetime
+        from flask_security import current_user
+        from manager_rest.storage import get_storage_manager, models
+        from manager_rest.manager_exceptions import ConflictError
+
+        sm = get_storage_manager()
+        names = [key for key in config_dict if not config_dict[key] is None]
```

Comment:
Can be `config_dict.keys()`, why would the value be `None`?
Maybe we want to update the value to `None`